packages/wrangler/src/dev/hotkeys.ts

```diff
@@ -59,6 +61,31 @@ export default function registerDevHotKeys(
 				await devEnv.teardown();
 			},
 		},
+		{
+			keys: ["r"],
+			// omitting the label means it won't be printed but is still enabled
+			// label: "rebuild container",
+			handler: async () => {
+				if (
+					!devEnv.config.latestConfig?.dev?.enableContainers ||
+					!devEnv.config.latestConfig?.containers
```

Comment:
Should it test the length instead - maybe factor that in a function and re-use at the multiple places where you check?